boost-maven-plugin/src/main/java/boost/project/LibertyBoostMojo.java

```diff
@@ -172,6 +225,49 @@ private ExecutionEnvironment getExecutionEnvironment() {
         return executionEnvironment(mavenProject, mavenSession, pluginManager);
     }
 
+    /**
+     * 
+     * @param boosterConfigurators
+     * @return
+     */
+    private List<String> getBoosterFeatureNames(List<BoosterPackConfigurator> boosterConfigurators) {
+			List<String> featureStrings = new ArrayList<String>();
+			for (BoosterPackConfigurator bpconfig : boosterConfigurators){
+				featureStrings.add(bpconfig.getFeatureString());
+			}
+			
+			return featureStrings;
+		}
+
+	private List<String> findJ2EEAppFeatureDependencies(MavenProject project) {
+		
+    	List<String> listOfDependencies = new ArrayList<String>();
+		getLog().debug("first lets see what dependencies we find");
+		for (Artifact artifact : mavenProject.getArtifacts()) {
+			getLog().debug("found this dependency, adding as a string -> " + artifact.getArtifactId());
+			listOfDependencies.add(artifact.getArtifactId());
+		}
+    	
+    	return listOfDependencies;
+	}
+
+    /**
+     * Invoke the liberty-maven-plugin to run the install-server goal
+     *
+     */
+    private void installOpenLiberty() throws MojoExecutionException {
```

Comment:
I think this method can be removed, since OL gets installed along with other liberty-maven goals.